### ISSUE TITLE
[Deno] Error While Installing /functions Package

### ISSUE THREAD

**Issue by aunyks** [donmccurdy/glTF-Transform#834]
**Describe the bug**
When running a Deno script that imports `@gltf-transform/functions` via `https://esm.sh/@gltf-transform/functions`, a `500` HTTP response is returned when it tries to install [`https://esm.sh/v107/ndarray-lanczos@0.2.0/deno/ndarray-lanczos.js`](https://esm.sh/v107/ndarray-lanczos@0.2.0/deno/ndarray-lanczos.js). 

The error is:
```
/* esm.sh - error */
throw new Error("[esm.sh] " + "resovleESModule: open /tmp/esm-build-a5562132cd84dbc9fc7056377a6f582714f47852-3a3152a6/node_modules/ndarray-lanczos/dist/ndarray-lanczos.modern.js: no such file or directory");
export default null;
```

I suspect this is related to the build configuration of [donmccurdy/ndarray-lanczos](https://github.com/donmccurdy/ndarray-lanczos).

**Steps To Reproduce**
1. Create a file `error-test.js` with this as the only line: `import * as f from 'https://esm.sh/@gltf-transform/functions'`
2. Run `deno run --allow-net error-test.js`

**Expected behavior**
An error-free install and script execution.

**Versions:**
 - `Deno`: 1.25.0
 - `gltf-transform`: 3.0.3

**Comment by donmccurdy** [comment]
Unfortunately there are a number of obstacles. I'd like to support Deno, but in practice it's pretty complicated. Certain decisions by the TypeScript team, and the difficulty of running the unit tests in multiple environments, have been obstacles. See #457.

I'll take a look at the ndarray issues, those may be simple enough to fix:

- https://github.com/donmccurdy/ndarray-lanczos/pull/103
- https://github.com/donmccurdy/ndarray-pixels/pull/119

But I'm not sure Deno will 'just work' even with those issues resolved. If you'd be up for doing some more investigation here, that'd be very welcome -- thanks!

**Comment by donmccurdy** [comment]
Possible fix, at least for the ndarray-related part of the problem:

- https://github.com/donmccurdy/glTF-Transform/pull/835

**Comment by aunyks** [comment]
Understood, if #835 doesn't fix this then I'll see if I can look into it some more. Thanks for the quick response anyway!

**Comment by donmccurdy** [comment]
Published v3.0.4 — Haven't tested thoroughly, but this much seems to work:

```js
import { Document, DenoIO } from 'https://esm.sh/@gltf-transform/core';
import { KHRONOS_EXTENSIONS } from 'https://esm.sh/@gltf-transform/extensions';
import { dedup } from 'https://esm.sh/@gltf-transform/functions';

const io = new DenoIO().registerExtensions(KHRONOS_EXTENSIONS);
const document = new Document();

console.log(document);
```

**Comment by aunyks** [comment]
Works for me as well! Thanks for addressing.
